AMW_angular/io/coding-guidelines.md

```diff
@@ -0,0 +1,51 @@
+# Coding Guidelines
+
+## Inject() over Constructor-Injections
+
+Use `inject()` instead of constuctor-injection to make the code more explicit and obvious.
+
+```typescript
+// use
+myService = inject(MyService);
+
+// instead of
+constructor(
+  private myservice: MyService
+) {}
+```
+
```

Comment:
## RxJS
Leverage RxJS for API calls, web sockets, and complex data flows, especially when handling multiple asynchronous events. Combine with Signals to simplify component state management.